docs/requirements.txt

```diff
@@ -9,3 +9,5 @@ pandas==1.4.2
 ray==1.13.0
 numpy
 git+https://github.com/charles9n/bert-sklearn.git@master
+shap==0.44.1
+xarray
```

Comment:
might be a good idea to set the version for xarray too

Reply:
I have included the exact version in setup.py as well.

Resolved by [a0d3f59](https://github.com/scikit-learn-contrib/hiclass/pull/106/commits/a0d3f59e05ce4e90eaeee2be791925b735a0f2b8).